components/director/internal/open_resource_discovery/global_registry.go

```diff
@@ -71,7 +71,7 @@ func (s *globalRegistryService) SyncGlobalResources(ctx context.Context) (map[st
 		return nil, errors.Wrapf(err, "while fetching global registry documents from %s", s.config.URL)
 	}
 
-	validationErrors, err := s.documentValidator.Validate(ctx, documents, s.config.URL, map[string]bool{}, docsString, "sap:base:v1")
+	validationErrors, err := s.documentValidator.Validate(ctx, documents, s.config.URL, map[string]bool{}, docsString, "sap:base:v1", "")
```

Comment:
Probably is good to introduce "Global" or similar namespace, instead using empty string. Otherwise it will be difficult to provide global ignore list.